test/utility/fragments.ts

```diff
@@ -1210,5 +1243,6 @@ export const fragments = {
   ceremony,
   partner,
   fundingAccount,
+  defaultMarketingRegion,
```

Comment:
I didn't know if this was needed here or not so I am happy to remove it if needed.

Reply:
Umm, I don't think it is.  Does something break without it?

Reply:
No, nothing breaks without it.